### DISCUSSION TITLE
Freeform 4 Release

### DISCUSSION THREAD

**Discussion by kmmartinez** [solspace/craft-freeform#392] [Questions & Answers] [answered]
Hi, 

Just wondering if you have a timeline of when you think Freeform 4 will officially be released for Craft 4 and no longer be in beta?

**Comment by kjmartens** [comment, marked as answer]
Hi @kmmartinez,

The beta is proving to be very stable so far, and more and more customers are using it, so we'll likely remove it from beta status within the next week or two. 🙂
